evap/results/exporters.py

```diff
@@ -107,6 +107,7 @@ def export(self, response, course_types_list, include_not_enough_answers=False,
                     used_questionnaires.add(questionnaire)
                 courses_with_results.append((course, results))
 
+            courses_with_results.sort(key=lambda cr: cr[0].name)
             courses_with_results.sort(key=lambda cr: cr[0].type)
```

Comment:
found this: http://stackoverflow.com/questions/4233476/sort-a-list-by-multiple-attributes
```
courses_with_results.sort(key=lambda cr: (cr[0].type, cr[0].name))
```